csharp/DEVELOPER.md

```diff
@@ -0,0 +1,81 @@
+# Developer Guide
+
+This document describes how to set up your development environment to build and test the GLIDE for Redis C# wrapper.
+
+### Development Overview
+
+The GLIDE C# wrapper consists of both C# and Rust code.
+
+### Build from source
+
+#### Prerequisites
+
+Software Dependencies
+
+-   .net sdk 6 or 8
+-   git
+-   GCC
+-   pkg-config
+-   protoc (protobuf compiler)
+-   openssl
+-   openssl-dev
+-   rustup
+
+**Dependencies installation for MacOS**
+
+visit https://dotnet.microsoft.com/en-us/download/dotnet
+to download .net 6 and 8 installer
```

Comment:
.net 6 and 8 installer -> .net installer

Reply:
Done